.circleci/config.yml

```diff
@@ -198,18 +202,18 @@ jobs:
 
       - restore_cache:
           name: restore mssql installation file
-          key: downloads-{{ checksum "test/bin/install-mssql.ps1" }}
+          key: downloads-<< parameters.mssql_version >>
```

Comment:
I recommend keeping the checksum, even if you want to also introduce the mssql version to the key

Reply:
added with aa830a48.